app/views/layouts/application.html.erb

```diff
@@ -6,10 +6,9 @@
 
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-
   <%= csrf_meta_tags %>
   <%= csp_meta_tag %>
-
+  <%= javascript_include_tag "rails-ujs" %>
```

Comment:
Hotwire Turbo is basically the new approach that is replacing UJS (unobtrustive javascript), so we shouldn't re-introduce this here...